kolibri/plugins/learn/assets/src/vue/card-grid/index.vue

```diff
@@ -1,7 +1,10 @@
 <template>
 
   <div>
-    <h3 v-if="header" class="header">{{header}}</h3>
+    <div class="header">
+      <h3 v-if="header">{{header}}</h3>
```

Comment:
Styling can be changed easily, bigger issue is incorrectly nested heading level...

But I agree, let's tackle that one in a separate PR.